cloud-optimized-geotiffs/accessing-cogs-in-r-terra.ipynb

```diff
@@ -0,0 +1,526 @@
+{
```

Comment:
<p>Let's note this is equivalent to using a random bounding box that intersects the tile.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>